packages/react/src/component-tests/IcBackToTop/IcBackToTop.cy.tsx

```diff
@@ -0,0 +1,77 @@
+/// <reference types="Cypress" />
+
+import React from "react";
+import { mount } from "cypress/react";
+import { NOT_BE_VISIBLE, CONTAIN_TEXT, BE_VISIBLE } from "../utils/constants";
+import { BackToTop } from "./IcBackToTopTestData";
+import { CYPRESS_AXE_OPTIONS } from "../../../cypress/utils/a11y";
+
+const DEFAULT_TEST_THRESHOLD = 0.08;
+
+describe("IcBackToTop", () => {
+  beforeEach(() => {
+    cy.injectAxe();
+  });
+
+  afterEach(() => {
+    cy.task("generateReport");
+  });
+
+  it("should be hidden", () => {
```

Comment:
This is a bit pedantic, but I think this test should have a more specific description. "should be hidden when viewport is at the top of the page"